src/components/Game.js

```diff
@@ -245,3 +246,7 @@ class Game extends Component {
 }
 
 export default Game;
+
+Game.propTypes = {
+  region: PropTypes.string.isRequired
```

Comment:
Can we add the same to the GameMap component? It's passed the region prop as well.

Reply:
Added, thanks!